README.md

```diff
@@ -134,13 +136,13 @@ struct Feature: ReducerProtocol {
         return .none
 
       case .numberFactButtonTapped:
-        return .task { [count = state.count] in 
+        return .task { [number = state.count] in
```

Comment:
Good catch! I would suggest we keep the `count` naming instead of introducing another name for the variable:

```suggestion
        return .task { [count = state.count] in
```